### PR TITLE
docs: update user guide and make find command docs consistent

### PR DESCRIPTION
**Pull request by ForAeons**
Resolves https://github.com/AY2324S2-CS2103T-F13-1/tp/issues/204

### CODE REVIEW

**Review by tituschewxj — CHANGES_REQUESTED** [review]
We should be consistent with the help window: `find KEYWORD [MORE_KEYWORDS]…​`, as we cannot enhance the text in the help window. 

image

In addition, we could add another info box in the UG on why we have two `KEYWORD` and `MORE_KEYWORDS` and give an explanation for each parameter.
* This is to provide greater **clarity**, as listing the parameter name twice in a single command format may cause confusion.
* `KEYWORD`: This is a required keyword
* `[MORE_KEYWORDS]...`: This is an optional field, that can accept more keywords.

**Comment by ForAeons** [comment]
> We should be consistent with the help window: `find KEYWORD [MORE_KEYWORDS]…​`, as we cannot enhance the text in the help window.
> 
> image
> 
> In addition, we could add another info box in the UG on why we have two `KEYWORD` and `MORE_KEYWORDS` and give an explanation for each parameter.
> 
>     * This is to provide greater **clarity**, as listing the parameter name twice in a single command format may cause confusion.
> 
>     * `KEYWORD`: This is a required keyword
> 
>     * `[MORE_KEYWORDS]...`: This is an optional field, that can accept more keywords.

Good point, updated

**Review by Anant1902 — APPROVED** [review]
LGTM

File: docs/UserGuide.md [hunk 1]
```diff
@@ -257,8 +257,16 @@ This command differs from most other commands that use the `NUSNET_ID` to identi
...
 
-Format: `find KEYWORD [MORE_KEYWORDS]`
+Format: `find KEYWORD [MORE_KEYWORDS]…​`
 
+<box type="info" seamless>
+
+**Parameters of find command:**
+
+- `KEYWORD` is the first keyword to search for. This is a compulsory parameter.
+- `[MORE_KEYWORDS]` are additional, optional keywords to search for. You can have any number of additional keywords.
```
Comment (tituschewxj) [inline docs/UserGuide.md:267]:
```suggestion
- `[MORE_KEYWORDS]...` are additional, optional keywords to search for. You can have any number of additional keywords.
```

File: docs/UserGuide.md [hunk 2]
```diff
@@ -257,8 +257,16 @@ This command differs from most other commands that use the `NUSNET_ID` to identi
 
 Finds students whose names contain any of the given keywords.
 
-Format: `find KEYWORD [MORE_KEYWORDS]`
+Format: `find KEYWORD [MORE_KEYWORDS]…​`
 
+<box type="info" seamless>
+
+**Parameters of find command:**
```
Comment (tituschewxj) [inline docs/UserGuide.md:264]:
```suggestion
**Parameters of `find` command:**
```

**Review by tituschewxj — CHANGES_REQUESTED** [review]
Just a few more nits.

In addition, can we add the following in the `**About the command format:**` box in the UG, maybe after the extraneous parameters part:
```
<box type="tip" seamless>

<span class="semi-bold">Each parameter will appear at most once in the command format.</span>

This provides greater clarity on how a command should be used.
</box>
```

**Review by tituschewxj — APPROVED** [review]
LGTM! Resolve the merge conflicts and we can merge this.

**Pull request merged**